src/component/Form.js

```diff
@@ -1,30 +1,35 @@
 import './styles/form.css';
-import { useDispatch, useSelector } from 'react-redux';
-import { useState } from 'react';
-import { addBook } from '../redux/book/booksSlice';
+import { useDispatch } from 'react-redux';
+import { useState, useEffect } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { addBooks, fetchBooks } from '../redux/book/booksSlice';
 
 export default function Form() {
   const [tit, setTitle] = useState('');
   const [auth, setAuthor] = useState('');
 
   const dispatch = useDispatch();
-  const { totalbooks } = useSelector((state) => state.book);
+  useEffect(() => {
+    dispatch(fetchBooks());
+  }, []);
   const book = () => {
-    const i = totalbooks.length + 1;
+    document.querySelector('.title').value = '';
+    document.querySelector('.author').value = '';
     const b = {
       title: tit,
       author: auth,
-      item_id: `item${i}`,
+      item_id: uuidv4(),
+      category: 'Economy',
     };
     return b;
   };
   return (
     <form>
```

Comment:
- [ ] 👍 Good progress! To match the design more closely, consider adding a vertical line before the form elements. Ensure that the line has the proper styling to align with the design. This addition will enhance the visual consistency of your project. Keep up the great work! 🌟📝